pkg/identityserver/store/gateway_antenna.go

```diff
@@ -29,21 +29,27 @@ type GatewayAntenna struct {
 	Gain float32
 
 	Location
+
+	Source int32 `gorm:"default:3 not null"`
```

Comment:
Using default value 3, which is `SOURCE_REGISTRY`, to preserve info for existing gateways.

The `Source` field itself probably belongs inside the `Location` type, but that breaks `EndDeviceLocation`:

https://github.com/TheThingsNetwork/lorawan-stack/blob/f62b09c8c06e46f8b26d9e9f757212a0f8f7cc9c/pkg/identityserver/store/end_device_location.go#L20-L29